tsstats/tests/test_ident_map.py

```diff
@@ -24,3 +25,19 @@ def test_ident_map(identmap_clients):
     assert clients['5'] == cl
     assert clients['UID1'] == uidcl
     assert clients['UID5'] == uidcl
+
+
+@pytest.fixture(scope="module")
```

Comment:
Additionally I don't think this should be fixture, as there's currently just one test-function using it.
(Yeah, I know, `identmap_clients` shouldn't be one then, either. I will fix that once I start refactoring some tests (especially template-tests have some more work to do))